submissions/Yasya23/holi-style-popup/style.css

```diff
@@ -0,0 +1,177 @@
+* {
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+img {
```

Comment:
please avoid styling with tags, it can lead to confusing results. you can move this property to `popup__img` class

Reply:
Property moved